create_or_update_mboms/bulk_create_or_update_mboms.py

```diff
@@ -29,13 +29,15 @@
 
 def create_or_update_mboms(api: Api, input_list: list[dict], is_level_notation: bool = False):
     """Create or update mboms."""
-    notation_var: bool = "levelInputs" if is_level_notation else "depthInputs"
+    notation_var: str = "levelInputs" if is_level_notation else "depthInputs"
+    empty_var: str = "levelInputs" if not is_level_notation else "depthInputs"
     request_body = {
         "query": queries.CREATE_OR_UPDATE_MBOMS,
         "variables": {
             "input": {
                 "importerType": "LEVEL" if is_level_notation else "DEPTH",
-                notation_var: input_list
+                notation_var: input_list,
+                empty_var: []
```

Comment:
Oh interesting, you have to pass an empty list for the type that you are not using?

Reply:
Yea you need to pass an empty list to at least one of the parameters. We should probably adjust that on the API side, but figured for now will at least fix this script.